app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailActivity.kt

```diff
@@ -66,7 +65,7 @@ class SessionDetailActivity :
         }
 
         binding.detailSessionsPager.adapter = pagerAdapter
-        drawerMenu.setup(binding.toolbar, binding.drawerLayout, binding.drawer)
+        drawerMenu.setup(null, binding.drawerLayout, binding.drawer)
```

Comment:
According to [this](https://github.com/DroidKaigi/conference-app-2018/pull/286/files#r162315318) , You can remove null parameter. 👍